provisioner/k8s/controllers/bundleinstance_controller.go

```diff
@@ -51,13 +48,13 @@ import (
 	"sigs.k8s.io/yaml"
 
 	olmv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
-	"github.com/operator-framework/rukpak/internal/convert"
 	helmpredicate "github.com/operator-framework/rukpak/internal/helm-operator-plugins/predicate"
 	"github.com/operator-framework/rukpak/internal/storage"
+	"github.com/operator-framework/rukpak/internal/util"
 )
 
 const (
-	registryV1ProvisionerID = "core.rukpak.io/registry+v1"
+	plainBundleProvisionerID = "core.rukpak.io/plain"
```

Comment:
Yep, this should just be a short term stop gap. It would be nice to also version this provisioner ID. I think another thing we'll need to think about is the domain we're using, and whether core.rukpak.io makes sense going forward.